projects/ngx-favicon/tsconfig.lib.json

```diff
@@ -17,6 +17,7 @@
       "@angular/*": ["../node_modules/@angular/*"]
     }
   },
+  "files": ["src/main.ts", "src/polyfills.ts"],
```

Comment:
Is this normal? I would've thought that the library shouldn't be aware of polyfills

Reply:
Yes, skipping this property produces an annoying CLI warning of `...part of the TypeScript compilation but it's unused` when running `serve` (at least after updating all the packages). This fix works. Probably there's any better solution but I haven't found any when was trying get rid of the warning.

Presumably, it would make sense to pay some more attention to ts configs and traverse all the properties to make sure there's no unnecessary stuff there after the update (or on contrary - no lack of properties for updated api). But I would suggest to make it separately.

Reply:
If you look here https://github.com/cloudnc/ngx-favicon/tree/master/projects/ngx-favicon/src

There's no `main.ts` nor `polyfills.ts` :thinking: 

![image](https://user-images.githubusercontent.com/4950209/92466132-f342ff80-f1cf-11ea-882a-bcd897814348.png)

So this looks weird to me

Reply:
You're right, my bad, I've just re-tested that without the unnecessary properties and it works fine now. Probably it was some TS glitch that I'd got at the moment.
I've pushed the update removing the `files` property from the lib config 👌